### PR TITLE
Add tests for ApiClient

### PR DESCRIPTION
**Pull request by asindarov**
I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

**Comment by Asadbek07** [comment]
> I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

**Comment by Asadbek07** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/exadel-inc/compreface-net-sdk/blob/develop/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> _**I have read the CLA Document and I hereby sign the CLA**_
> 
> You can retrigger this bot by commenting **recheckcla** in this Pull Request



> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/exadel-inc/compreface-net-sdk/blob/develop/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> _**I have read the CLA Document and I hereby sign the CLA**_
> 
> You can retrigger this bot by commenting **recheckcla** in this Pull Request



> **recheckcla**



> I have read the CLA Document and I hereby sign the CLA



> I have read the CLA Document and I hereby sign the CLA

recheckcla

**Review by Evgeniy-Sinicin — DISMISSED** [review]
Everything seems to be fine 👍

**Comment by asindarov** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by vldmrklnk — DISMISSED** [review]

**Comment by Asadbek07** [comment]
recheckcla

**Comment by asindarov** [comment]
recheckcla

**Comment by Evgeniy-Sinicin** [comment]
Abracadabra!!! ✨

**Comment by asindarov** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by vldmrklnk — DISMISSED** [review]

**Review by Evgeniy-Sinicin — DISMISSED** [review]

**Review by vldmrklnk — DISMISSED** [review]

File: Exadel.Compreface.UnitTests/ApiClient/ApiClientTests.HttpVerbs.cs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+namespace Exadel.Compreface.UnitTests.ApiClient;
+
+public partial class ApiClientTests
+{
+    [Fact]
+    public async Task GetJsonAsync_ShouldUseGETHttpMethod()
+    {
+        // simulating api calls here and constructing fake response
+        _httpTest.RespondWith(SuccessMessage, 200);
```
Comment (Evgeniy-Sinicin) [inline Exadel.Compreface.UnitTests/ApiClient/ApiClientTests.HttpVerbs.cs]: Changed 400 to BadRequestStatusCode, but 200 is not 😄

**Review by Evgeniy-Sinicin — DISMISSED** [review]

File: Exadel.Compreface.UnitTests/Services/SubjectExampleServiceTests.cs [hunk 2]
```diff
@@ -60,7 +60,7 @@ public async Task AddExampleSubjectAsync_TakesRequestModel_ReturnsProperResponse
     }
 
     [Fact]
-    public async Task AddBase64ExampleSubjectAsync_TakesRequestModel_ReturnsProperResponseModel()
+    public async Task AddBase64SubjectExampleAsync_TakesRequestModel_ReturnsProperResponseModel()
```
Comment (Ukrainiangirl) [inline Exadel.Compreface.UnitTests/Services/SubjectExampleServiceTests.cs:63]: this test doesn't work

Expected: Exadel.Compreface.DTOs.ExampleSubjectDTOs.AddBase64ExampleSubject.AddBase64ExampleSubjectResponse
Actual:   (null)

Comment (asindarov) [reply Exadel.Compreface.UnitTests/Services/SubjectExampleServiceTests.cs:63]: I did not changed it, someone accidentally commited to my PR

Comment (asindarov) [reply Exadel.Compreface.UnitTests/Services/SubjectExampleServiceTests.cs:63]: I found the problem and fixed it

File: Exadel.Compreface.UnitTests/ApiClient/ApiClientTests.HttpVerbs.cs (hunk 1 above)
Comment (asindarov) [reply Exadel.Compreface.UnitTests/ApiClient/ApiClientTests.HttpVerbs.cs]: Yeah, I forgot to do it. Now, I have added SuccessResponseStatusCode field

**Review by Ukrainiangirl — APPROVED** [review]

**Pull request merged**